app/models/content/content.ts

```diff
@@ -88,7 +88,13 @@ export const ContentModel = types
       return self.creator?.normalizedName || ""
     },
     get normalizedTitle() {
-      return self.title || decodeURI(self.url).split("?")[0]
+      let url: string
+      try {
+        url = decodeURI(self.url).split("?")[0]
+      } catch (error) {
+        logError(error)
+      }
+      return self.title || url
```

Comment:
How about we add a shortcut case for `if (self.title) return self.title`, without trying to decode URI?